core/jobs.py

```diff
@@ -875,10 +1120,9 @@ def _switch_active_realtime_class_transactional():
             _switch_active_realtime_class_transactional)
 
     @classmethod
-    def _clear_inactive_realtime_layer(
-            cls, latest_created_on_datetime):
+    def _clear_inactive_realtime_layer(cls, latest_created_on_datetime):
         """Deletes all entries in the given realtime datastore class whose
```

Comment:
Hm, the old docstring was wrong: there is no "given realtime datastore class". Instead:

Deletes all entries in the inactive realtime layer whose ...